templates/resources.hbs

```diff
@@ -1,37 +1,39 @@
-<div class="resources-inner" role="complementary">
+{{! make the _globals object in course.json available to this template}}
+{{import_globals}}
+<div class="resources-inner">
+	{{a11y_aria_label _globals._extensions._resources.resources}}
```

Comment:
Do we have PR pertaining to this change in framework?
The change should be at line https://github.com/adaptlearning/adapt_framework/blob/master/src/course/en/course.json#L80

Reply:
Not yet. I will update it. I've got to fix a bug with the grunt files first